validation/src/test/scala/hmda/validation/rules/lar/validity/V360Spec.scala

```diff
@@ -0,0 +1,37 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Denial, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V360Spec extends LarEditCheckSpec {
+  property("Denials with all different values must pass") {
+    forAll(larGen) { lar =>
+      val goodDenial = lar.denial.copy(reason1 = "1", reason2 = "2", reason3 = "")
```

Comment:
I'd like to see this test be a bit more generic. For instance, you could use a generator like `badOptionGen` below to generate three values, then just use a `whenever` clause to make sure they're all different.